source/tutorial/deploying.md

```diff
@@ -26,28 +47,23 @@ Then you can use the `surge` command to deploy your application.
 Note you will also need to rename index.html to 200.html to enable Ember's client-side routing.
 
 ```shell
-ember build --environment=development
 mv dist/index.html dist/200.html
 surge dist funny-name.surge.sh
 ```
 
-We chose funny-name.surge.sh but you may use any unclaimed subdomain you like or 
+We chose funny-name.surge.sh but you may use any unclaimed subdomain you like or
 use a custom domain that you own and have pointed the DNS to one of surges servers.
 If the second argument is left blank surge will prompt you with a suggested subdomain.
 
-To deploy to the same URL after making changes, perform the same steps, reusing 
+To deploy to the same URL after making changes, perform the same steps, reusing
 the same domain as before.
 
 ```shell
 rm -rf dist
-ember build --environment=development
```

Comment:
oops, gotta clean this up first. to re-run the build (with the key)

Reply:
PR not ready?

Reply:
yeah, just need to add back build with key here.